app/controllers/memberships_controller.rb

```diff
@@ -4,34 +4,50 @@ def show
     response = conn.get('/api/magic_link/login', { token: params[:id] }, headers)
 
     if response.status != 200
-      # raise
       redirect_to not_found_path
     else
       data = JSON.parse(response.body, symbolize_names: true)
```

Comment:
Maybe that can go into a follow up PR but we also need to check the existing bids contained in that response. If there is already a bid placed for the relevant timespan (1.4.2022 - 31.3.2023) we need to show an info to the user that the bid was already placed and deny further bidding. Maybe for convenience we show in that case the info we have at hand. Like was was bid and when it was placed.

Reply:
For the sake of simplicity I think we can rely on checking if there is a bid starting at 1.4.2022.